tools/populate-owners/main.go

```diff
@@ -110,6 +115,107 @@ func (orgRepo *orgRepo) getDirectories(dirs ...string) (err error) {
 }
 
 func (orgRepo *orgRepo) getOwners() (err error) {
+	err = orgRepo.getOwnersHTTP()
+	if err == nil {
+		return nil
+	}
+	fmt.Fprintf(os.Stderr, "%v\n", err)
+
+	return orgRepo.getOwnersGit()
+}
+
+// getOwnersHTTP is fast (just the two files we need), but only works
+// on public repos unless you have an auth token.
+func (orgRepo *orgRepo) getOwnersHTTP() (err error) {
+	commitURI := fmt.Sprintf("https://api.github.com/repos/%s/%s/commits/HEAD", orgRepo.Organization, orgRepo.Repository)
```

Comment:
I'm fairly certain from prior tests that if this is run from inside of either an office block or from the cloud the IPs involved will have exhausted their quota at all times

Reply:
> I'm fairly certain from prior tests that if this is run from inside of either an office block...

This code is now coming in via #1791, and I've mentioned this concern in the topic post there.